src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/StringSearchParam.sql

```diff
@@ -7,7 +8,11 @@
     TextOverflow nvarchar(max) COLLATE Latin1_General_100_CI_AI_SC NULL,
     IsHistory bit NOT NULL,
     IsMin bit CONSTRAINT string_IsMin_Constraint DEFAULT 0 NOT NULL,
-    IsMax bit CONSTRAINT string_IsMax_Constraint DEFAULT 0 NOT NULL
+    IsMax bit CONSTRAINT string_IsMax_Constraint DEFAULT 0 NOT NULL,
+    TextHash varchar(64) COLLATE Latin1_General_100_CI_AI_SC NOT NULL,
```

Comment:
>varchar [](http://example.com/codeflow?start=13&length=7)

Since it has constant length, would it make sense to make it char(64).

Ideally I would prefer to have binary(32) to remove any string allocations and just pass byte array back and forth.

Reply:
Good point, updated.